### ISSUE TITLE
Amazing job

### ISSUE THREAD

**Issue by Killaheala** [briansd9/exile-diary#21]
Hello, i became a fan of this and thanks for doing it, ill patronage as well, just wonder if you could maybe do an infinite list of all the maps im doing, it seems like its showing the last 11 maps you do, id like all the maps, or maybe theres the option.

Good job again, keep the good work.

**Comment by briansd9** [comment]
Thank you very much for the support! Greatly appreciated.

The infinite list is there already-- when you have a map displayed, there's a dropdown list at the top that you can pick from, or you can use the left and right arrow keys to go to the next or previous map.

Or if you want to see them all at the same time, go to the "search" option in the sidebar and just click search without entering any filters.

**Comment by Killaheala** [comment]
Yeah i can see it :D thanks, anyway is the "Kills" working? it doesnt show any kill i do or it has to be pvp?

**Comment by ThiagoCururu** [comment]
I was going to create a "issue" just to tell you and everyone who reads it, how amazing your software is! But as I don't like spam and I was afraid you'll be startled with a notification of a new issue, I'll leave my humble THANK YOU here instead.

Amazing job!
